rasa/core/channels/webexteams.py

```diff
@@ -26,21 +26,21 @@ def __init__(self, access_token, room):
     ) -> None:
         recipient = self.room or recipient_id
         for message_part in text.split("\n\n"):
-            self.api.messages.create(roomId=recipient, text=message_part)
-
+            self.api.messages.create(roomid=recipient, text=message_part) if self.room else self.api.messages.create(toPersonId=recipient, text=message_part)
     async def send_image_url(
         self, recipient_id: Text, image: Text, **kwargs: Any
     ) -> None:
         recipient = self.room or recipient_id
-        return self.api.messages.create(roomId=recipient, files=[image])
+        return self.api.messages.create(roomid=recipient, files=[image]) if self.room else self.api.messages.create(toPersonId=recipient, files=[image])
```

Comment:
Same here:
```python
        if self.room:
            return self.api.messages.create(roomid=recipient, files=[image])
        else:
            return self.api.messages.create(toPersonId=recipient, files=[image])
```

Reply:
also, can you check to make sure if it is `roomId` or `roomid` please? looks like the capitalization got changed in the edit